0003-interledger-protocol/0003-interledger-protocol.md

```diff
@@ -316,6 +316,20 @@ Relative errors indicate that the payment did not have enough of a margin in ter
 | **R02** | **Insufficient Timeout** | The connector could not forward the payment, because the timeout was too low to subtract its safety margin. The sender MAY try again with a higher expiry, but they SHOULD NOT do this indefinitely or a malicious connector could cause them to tie up their money for an unreasonably long time. |
 | **R99** | **Application Error** | Reserved for application layer protocols. Applications MAY use names other than `Application Error`. |
 
+### ILP Fulfillment Format
+
+Here is a summary of the fields in the ILP fulfillment format:
```

Comment:
Where is the fulfillment itself? When you say "ILP fulfillment format", I expect to see 32 bytes

Reply:
I didn't include the fulfillment since this is the ILP packet that travels *with* the fulfillment. Similar to how the payment packet doesn't contain the condition, because it travels *with* the condition.

I actually now agree with @adrianhopebailie that the condition probably should have been part of the ILP packet, but hindsight is 20/20 and I'd rather keep things consistent.

If we include the fulfillment in the ILP packet and we do this as a non-breaking change (add new `ilp` parameter in LPI `fulfillCondition`), then we'd be sending the fulfillment twice.

Agree, that the above rationale is not obvious, so I can add some verbiage to explain why this is the way it is.